.github/workflows/gradle.yml

```diff
@@ -0,0 +1,36 @@
+name: Gradle Build and Test
+
+on: [push, pull_request]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up JDK 11
+        uses: actions/setup-java@v2
+        with:
+          java-version: '11'
+          distribution: 'adopt'
+
+      - name: Run Gradle Build
+        run: ./gradlew build
```

Comment:
This will run tests as well. Using this command `./gradlew build -x test` will not run tests until the next step which is what I think we want

Reply:
You're right, no need to attempt running the tests twice, and this'll provide more info about the tests during the test step not build. I'll update.